main/utils.py

```diff
@@ -144,3 +145,22 @@ def get_month_dates_for_previous_year() -> list[tuple[date, date]]:
 
     dates.reverse()
     return dates
+
+
+def order_queryset_by_property(  # type: ignore[explicit-any]
+    queryset: QuerySet[Any], property: str, is_descending: bool
+) -> QuerySet[Any]:
+    """Orders a queryset according to a specified property."""
```

Comment:
I would add the description of the input arguments and return value, as well as a short explanation of how this is working. It is pretty advanced django stuff, so some extra help for the reader would come handy.